src/D2L.Bmx/Aws/AwsRole.cs

```diff
@@ -0,0 +1,7 @@
+namespace D2L.Bmx.Aws;
+
+public struct AwsRole {
```

Comment:
let's make it a record with a primary constructor like the rest of our new C# code.
also - make it internal